v-next/core/test/internal/fixture-plugins/config-plugin.ts

```diff
@@ -0,0 +1,22 @@
+import type { HardhatUserConfig } from "../../../src/types/config.js";
```

Comment:
Just to double check: this import and the following one are both coming from the "core" package.
They are imported in different ways on purpose? One via relative path and one via "@nomicfoundation/hardhat-core"

Reply:
Good catch. I have opted to make both imports relative, rather than mixing.